package.json

```
@@ -1,17 +1,28 @@
 {
   "name": "@mtvh/mtvh-formio",
-  "version": "2.0.12",
+  "version": "2.0.10",
```

Should we bump the version?
```suggestion
  "version": "2.1.0",
```